client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -265,11 +281,12 @@ const percentCartModel = {
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
 			couponCode: couponCodeField,
-			percentDiscount: {
-				component: PercentField,
-				labelText: translate( 'Percent Cart Discount', { context: 'noun' } ),
-				isRequired: true,
-			},
+			percentDiscount: (
+				<PercentField
+					labelText={ translate( 'Percent Cart Discount', { context: 'noun' } ) }
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.